File: Scripts/GlobalAmmo.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.UI;

public class GlobalAmmo : MonoBehaviour {
	public static int CurrentAmmo = 0;
	public int InternalAmmo;
	public GameObject AmmoDisplay;

	void Update () {
		InternalAmmo = CurrentAmmo;
		AmmoDisplay.GetComponent<Text>().text = "" + InternalAmmo;
	}
}

File: Scripts/GlobalAmmo.js
<<<<<<< HEAD
﻿import UnityEngine.UI;

static var CurrentAmmo : int;
var InternalAmmo : int;
var AmmoDisplay : GameObject;

static var LoadedAmmo : int;
var InternalLoaded : int;
var LoadedDisplay : GameObject;

function Update () {
	InternalAmmo = CurrentAmmo;
	InternalLoaded = LoadedAmmo;
	// Can't  be text = InternalAmmo
	AmmoDisplay.GetComponent.<Text>().text = "" + InternalAmmo;
	LoadedDisplay.GetComponent.<Text>().text = "" + InternalLoaded;
}
=======
﻿import UnityEngine.UI;

static var CurrentAmmo : int;
var InternalAmmo : int;
var AmmoDisplay : GameObject;

static var LoadedAmmo : int;
var InternalLoaded : int;
var LoadedDisplay : GameObject;

function Update () {
	InternalAmmo = CurrentAmmo;
	InternalLoaded = LoadedAmmo;
	// Can't  be text = InternalAmmo
	AmmoDisplay.GetComponent.<Text>().text = "" + InternalAmmo;
	LoadedDisplay.GetComponent.<Text>().text = "" + InternalLoaded;
}
>>>>>>> 416bb19edcc99a854f8e26b06ded8d914e8b3eae
